views/add_event.hbs

```diff
@@ -0,0 +1,111 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label> Image URL: </label>
+    <input class="form-control" type="text" name="imageUrl" placeholder="image URL link">
+
+    <label>Start Date:</label>
+    <input class="form-control" type="date" name="startDate" placeholder="example yyyy-mm-dd" required>
+
+    <label>End Date:</label>
+    <input class="form-control" type="date" name="endDate" placeholder="example yyyy-mm-dd" required>
+
+    <label>Start Time: </label>
+    <input class="form-control" type="text" name="startTime" placeholder="example 00:00" required>
+
+    <label>End Time: </label>
+    <input class="form-control" type="text" name="endTime" placeholder="example 00:00" required>
+
+    <label>Place: </label>
+    <li id="addPlaceIcon" class="glyphicon glyphicon-plus"></li><span class="addPlaceSpan text-muted">add new place</span>
+    <div id="option-list-container"></div>
+    <select id="selectAllPlaces" class="options form-control" name="placeId" required>
```

Comment:
Could you add an empty line before and after this select element? (since the rest of the file is so nicely spaced out)